tests/testthat/test-epinow.R

```diff
@@ -124,10 +124,8 @@ test_that("epinow runs without error when saving to disk", {
 
 test_that("epinow can produce partial output as specified", {
   out <- suppressWarnings(epinow(
-    reported_cases = reported_cases,
-    generation_time = generation_time_opts(
-      example_generation_time, weight_prior = FALSE
-    ),
+    data = reported_cases,
+    generation_time = generation_time_opts(example_generation_time),
```

Comment:
why remove `weight_prior = FALSE`?

Reply:
Wasn't intentional. Not sure how that happened 🤦🏾‍♂️